app/models/department.go

```diff
@@ -7,9 +7,9 @@ import (
 )
 
 type Department struct {
-	ID              uuid.UUID         `json:"id" db:"id"`
-	Name            string            `json:"name" db:"name"`
-	Description     string            `json:"description" db:"description"`
-	CreatedAt       time.Time         `json:"created_at" db:"created_at"`
-	UpdatedAt       time.Time         `json:"updated_at" db:"updated_at"`
-}
\ No newline at end of file
+	ID          uuid.UUID `db:"id"`
+	Name        string    `db:"name" fako:"first_name"`
+	Description string    `db:"description" fako:"paragraph"`
```

Comment:
Why "first_name", "paragraph"? Please remove or rename them

Reply:
First_name was implemented because this returns me the first name of a person randomly and paragraph because this returns me a paragraph which is what I need in those fields, attached documentation: https://github.com/wawandco/fako

Reply:
Oh gotcha! Thanks for the explanation.